qt_ui/windows/newgame/QNewGameWizard.py

```diff
@@ -672,7 +672,9 @@ def __init__(self, parent=None):
 
         modLayout = QtWidgets.QGridLayout()
         modLayout_row = 1
-        modLayout.addWidget(QtWidgets.QLabel("A-4E Skyhawk"), modLayout_row, 0)
+        modLayout.addWidget(
+            QtWidgets.QLabel("A-4E Skyhawk (version 2.0.0)"), modLayout_row, 0
+        )
```

Comment:
just a side note: having the version number coded in the modded aircraft py extension would be better but leaving it here is also okay for the time being.
Something we should add with another Mod Support functionaliity. I think more about a system which does not require anything to be changed in the main Liberation code at all (like with the plugin system). Mod support is a time consuming thing to bring into liberation. Would be good to make it independent so that mod support developers can provide support packages independant -> No work to be done from liberation side later = no updates for mod updates required.

Reply:
#2178